File: JS/divby3or5.js

function multiplesOf3Or5(number) {
let arr = []
let sum = 0
  for(let i = 3; i < number; i++) {
    if(i%3 == 0 || i%5 == 0){
      arr.push(i)
    }
  }
  //console.log(arr)

  for(let j = 0; j < arr.length; j++) {
    sum += arr[j];
  }
  console.log(sum)
 
}

multiplesOf3Or5(10);

File: Python/divBy3Or5.py
def multiplesOf3Or5(number):
    ans = []
    sum = 0

    for i in range(3, number):
        if i % 3 == 0 or i % 5 == 0:
            ans.append(i)

    for i in ans:
        sum += i

    print(sum)

multiplesOf3Or5(10)